install.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
+
+# CHECKS FOR SUDO ACCESS (added because of those who have seperate partitions for /home and /root) 
+
+if [ "$EUID" -ne 0 ]
+  then echo "Please don't run as root" && exit
+  exit
+fi
+
+# MAKE DIRECTORY SO THAT YOUR HOME DIRECTORY DOESN'T GET DIRTY
+
+cd /home/$USER
+mkdir -p junk-drawer
+cd junk-drawer
```

Comment:
this could be replaced with
```suggestion
cd $(mktemp -d)
```

Reply:
As i've said before, the reason i didn't use /tmp is because a lot of people use cron jobs to clear it out, which could interfere with the installation

Reply:
> this could be replaced with

As i've said before, the reason i didn't use /tmp is because a lot of people use cron jobs to clear it out, which could interfere with the installation

Reply:
The directory wouldn't have to persist, right? It's only going to take a few minutes tops with a bad connection.
The current step would fail for me, since my home directory is not writable.

Another option would be to use
```bash
TMPDIR="${XDG_CACHE_HOME:-$HOME/.cache}"
mkdir -p "$TMPDIR"
cd $(mktemp -d --tmpdir="$TMPDIR")
```

Reply:
> The directory wouldn't have to persist, right? It's only going to take a few minutes tops with a bad connection. The current step would fail for me, since my home directory is not writable.
> 
> Another option would be to use
> 
> ```shell
> TMPDIR="${XDG_CACHE_HOME:-$HOME/.cache}"
> mkdir -p "$TMPDIR"
> cd $(mktemp -d --tmpdir="$TMPDIR")
> ```

I know some people who use 2 minute intervals to clean it out :skull:

Reply:
i agree with using `/tmp` for this. it's a very common pattern in shell scripts, and it's exactly what `/tmp` is intended to be used for.

having `/tmp` constantly wiped like that is going to break a lot more than just the catppuccin steam install script. we shouldn't strive to support such broken setups.

Reply:
Good point

Reply:
Lemme go fix it up real quick

Reply:
done